### ISSUE TITLE
Declarations of the round and the trunc functions are duplicated

### ISSUE THREAD

**Issue by artemiuss** [orafce/orafce#75]
> **float** with no precision specified is taken to mean **double precision**

[https://www.postgresql.org/docs/9.0/static/datatype-numeric.html](url)

```
CREATE OR REPLACE FUNCTION oracle.round(double precision, int)
RETURNS numeric
AS $$SELECT pg_catalog.round($1::numeric, $2)$$
LANGUAGE sql IMMUTABLE STRICT;

CREATE OR REPLACE FUNCTION oracle.round(float, int)
RETURNS numeric
AS $$SELECT pg_catalog.round($1::numeric, $2)$$
LANGUAGE sql IMMUTABLE STRICT;

```
```
CREATE OR REPLACE FUNCTION oracle.trunc(double precision, int)
RETURNS numeric
AS $$SELECT pg_catalog.trunc($1::numeric, $2)$$
LANGUAGE sql IMMUTABLE STRICT;

CREATE OR REPLACE FUNCTION oracle.trunc(float, int)
RETURNS numeric
AS $$SELECT pg_catalog.trunc($1::numeric, $2)$$
LANGUAGE sql IMMUTABLE STRICT;
```

**Comment by okbob** [comment]
should be fixed by commit 42b7f4fd1a04c634ca3e92efdb9963f0a33be74f

**Comment by okbob** [comment]
Thank you for report
